labs/b9.md

```diff
@@ -1,159 +1,539 @@
 ---
-title: Lab 9 - Version Control and Backups
+title: Lab 9 - Version Control and Backups 
 layout: lab
 ---
 
-## Git IRL
+In this lab, we'll be learning how to use Git and typical best practices for
+version control!
+
+### Pulling the code
+
+If you haven't already cloned the decal-lab git repository, run: `git clone
+https://github.com/0xcf/decal-labs.git`
+
+If you have already cloned the repository, go to your `decal-labs` directory and
+run `git pull` to pull the latest changes. The materials for this part of the
+lab will be in the `b9` directory.
+
+### Part 1: Git Basics
+
```

Comment:
Insert a super short blurb reminding people of what git is, maybe link to an intro resource or lecture

Reply:
I'll probably make another PR to add the lecture link once it's been uploaded.